### ISSUE TITLE
[simclasses] Failing HitMerging Tests (Trac #1618)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1610]
1_MCPEMergingSortingMerging
2_MCPulseMergingSortingMerging

Both fail.  I'm disabling both for now.  Both should be enabled again when they pass.

Migrated from https://code.icecube.wisc.edu/ticket/1618
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:45", 
    "description": "1_MCPEMergingSortingMerging\n2_MCPulseMergingSortingMerging\n\nBoth fail.  I'm disabling both for now.  Both should be enabled again when they pass.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067105393059", 
    "component": "combo simulation", 
    "summary": "[simclasses] Failing HitMerging Tests", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-04-02T01:10:33", 
    "milestone": "", 
    "owner": "benedikt.riedel", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160402T02:45:21: 
* benedikt.riedel commented:

> In changeset 144182:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="144182"
> Fixed tests. Closes #1610
> ```


* benedikt.riedel changed status from "new" to "closed"
* benedikt.riedel changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:45: 
* olivas commented:

> Milestone IceSim 5.0 deleted


* olivas changed milestone from "IceSim 5.0" to ""
